docs-site/docs/node-operators/getting-started.md

```diff
@@ -51,8 +44,7 @@ Building from source requires a few packages to be installed on your system. On
 apt install libssl-dev protobuf-compiler cmake make pkg-config llvm clang
 ```
 
-On other linux distros or Mac, use your package manager's equivalents.
-
+On other Linux distributions or MacOS, use your package manager's equivalents.
```

Comment:
it's macOS

![Screenshot 2023-12-31 at 14.29.15.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/6PLX7gYqg3XdX0xjUaLN/9aa3943f-35ec-49b3-a390-3706e904bb17.png)